symforce/codegen/template_util.py

```diff
@@ -45,8 +47,8 @@ def from_extension(extension: str) -> FileType:
             return FileType.LCM
         elif extension == "Makefile":
             return FileType.MAKEFILE
-        elif extension == "ts":
-            return FileType.TYPESCRIPT
+        elif extension == "js":
+            return FileType.JAVASCRIPT
```

Comment:
Probably leave typescript in here, unless we also delete it from the enum above?